packages/integrations/partytown/src/index.ts

```diff
@@ -0,0 +1,33 @@
+import type { AstroConfig, AstroIntegration } from 'astro';
+import sirv from './sirv.js';
```

Comment:
Yes! see the comment in `sirv.ts`:

```
// TODO: The below has been modified from the original sirv package to support 
// the feature of mounting the served files from a certain path (in this case, `/~partytown/`)
// It would be good to bring this into Astro for all integrations to take advantage of,
// and potentially also to respect your config automatically for things like `base` path.
```

This is a fairly common use-case afaict by looking through the Nuxt ecosystem, so I think adding something into Astro for all integration authors to use would be 👍